management-api-for-apache-cassandra/upgrade-deps.patch

```diff
@@ -0,0 +1,55 @@
+From ced3dc07c8a3cfa3277b4088b102fc05c2a343f4 Mon Sep 17 00:00:00 2001
+From: Batuhan Apaydin <batuhan.apaydin@chainguard.dev>
+Date: Sun, 14 Jan 2024 20:08:54 +0300
+Subject: [PATCH] upgrade deps
+
+Signed-off-by: Batuhan Apaydin <batuhan.apaydin@chainguard.dev>
+---
+ management-api-server/pom.xml | 4 ++--
+ pom.xml                       | 6 +++---
+ 2 files changed, 5 insertions(+), 5 deletions(-)
+
+diff --git a/management-api-server/pom.xml b/management-api-server/pom.xml
+index f2b9659..f638e0d 100644
+--- a/management-api-server/pom.xml
++++ b/management-api-server/pom.xml
+@@ -17,10 +17,10 @@
+   <artifactId>datastax-mgmtapi-server</artifactId>
+   <properties>
+     <rsapi.version>2.1.1</rsapi.version>
+-    <guava.version>30.1.1-jre</guava.version>
++    <guava.version>32.0.0-android</guava.version>
```

Comment:
Any specific reason for using the `-android` variant of Guava here? If this used `-jre` originally we should probably keep the same variant.

Reply:
there was a CVE reported and the report says it fixed in `32.0.0-android` this is why I used this version